tensorflow/python/training/slot_creator_test.py

```diff
@@ -136,6 +136,7 @@ def testCreateSlotFromVariableRespectsScope(self):
         slot = slot_creator.create_slot(v, v.initialized_value(), name="slot")
         self.assertEqual("scope/scope/var/slot", slot.op.name)
 
+  @test_util.run_v1_only("b/120545219")
```

Comment:
How about a note "tests v1 APIs" rather than that bug number? The bug is a note to triage tests that were failing when we enabled v2 builds; we shouldn't tag new tests that way.

Reply:
Sorry, I was confused with UT syntactic sugar.
Change to '@test_util.run_deprecated_v1', right?

Reply:
I think it needs a reason; replacing the "b/..." string with "tests v1 APIs" or something like that is fine. It's for humans, not test infrastructure.

Reply:
change to "run_deprecated_v1", only run this TestCase in graph mode